cfme/tests/configure/test_timeprofile.py

```diff
@@ -53,45 +54,39 @@ def test_timeprofile_name_max_character_validation():
 
 
 @pytest.mark.sauce
-def test_days_required_error_validation():
+def test_days_required_error_validation(appliance):
     tp = st.Timeprofile(
         description='time_profile' + fauxfactory.gen_alphanumeric(),
         scope='Current User',
         timezone="(GMT-10:00) Hawaii",
         days=False,
         hours=True)
-    if version.current_version() > "5.7":
-        tp.create(cancel=True)
-        assert "At least one day needs to be selected" == \
-               tp.timeprofile_form.days.angular_help_block
-        if version.current_version() > "5.8":
-            assert form_buttons.save.is_dimmed
-        else:
-            assert form_buttons.add.is_dimmed
-    else:
-        with error.expected("At least one Day must be selected"):
-            tp.create()
+    view = navigate_to(st.Timeprofile, 'All')
+    tp.create(cancel=True)
+    help_blocks = appliance.browser.widgetastic.elements("//span[contains(@class,"
```

Comment:
I am not sure how to do that because I am just registering it as an element which doesn't have any widget available for it. Is there something I am missing here?

Reply:
@anurag03 looks like this should be a Text widget:
https://github.com/RedHatQE/widgetastic.core/blob/master/src/widgetastic/widget.py#L895

you can pass the locator to that and then read the text from it.